docs_sphinx/advanced/custom_events.rst

```diff
@@ -27,23 +84,52 @@ model::
 
     event_mon = EventMonitor(group, 'custom_event', variables['var1', 'var2'])
```

Comment:
I know you didn't write this sentence but "it can also record other variables of the" should probably mention *"at the time of the event"*, no?